source/modulo_components/include/modulo_components/ComponentInterface.hpp

```diff
@@ -413,6 +413,14 @@ class ComponentInterface : public NodeT {
    */
   rcl_interfaces::msg::SetParametersResult on_set_parameters_callback(const std::vector<rclcpp::Parameter>& parameters);
 
+  /**
+   * @brief Parameter validation wrapper that validates the period and calls the validation function of the derived
+   * Component classes.
```

Comment:
I prefer to put specific behaviours in the details  
```suggestion
   * @brief Parameter validation wrapper
   * @details This validates the period and calls the validation function of the derived Component classes.
```